### PR TITLE
Fixes not indicating offline friends (#950)

### LINKED ISSUE #950: Player Indicators - Not Showing Offline Friends
As the title stats, for some odd reason, `Player#isFriend` is not returning true if the player **is on your friends list** and **offline**.

### PR DESCRIPTION
**Pull request by Infinitay**
Fixes #950 

Plugin previously used `Player#isFriend`, which would return false if the friend was offline. Using `Client#isFriended`, passing the parameter `false` to `mustBeLoggedIn` fixes this and allows for offline friends to be indicated.

### CODE REVIEW

**Comment by deathbeam** [comment]
Can you test this with people with spaces in their names? Also, marked this as feature request, because the previous behaviour was correct as well.

**Comment by Adam-** [comment]
Do we want this showing friends who are offline? Seems maybe OP

**Comment by rune3132** [comment]
Osbuddy does it, so it wouldn't be a problem from Jagex's perspective.

**Comment by Kamielvf** [comment]
Other clients don't check if your friends are online either

**Comment by deathbeam** [comment]
Still, I don't like it very much, for reasons I stated before. Only because OSB does it, should we too? Idk about that.

**Comment by SomeoneWithAnInternetConnection** [comment]
If "offline" is equivalent to "has PMs off/on friends only" it seems kinda ehhh, even if other clients do it. Both from a "maybe this is OP for PKing" sense, but it also just kinda feels slimy

**Comment by deathbeam** [comment]
After previous discord discussion the decision on this is to respect the game settings, and this PR became mess anyway.

File: runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsOverlay.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public Dimension render(Graphics2D graphics, Point parent)
 					renderPlayerOverlay(graphics, player, config.getOwnNameColor());
 				}
 			}
-			else if (config.drawFriendNames() && player.isFriend())
+			else if (config.drawFriendNames() && client.isFriended(player.getName(), false))
```
Comment (zeronuclear) [inline runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsOverlay.java]: Hello i have no idea how to comment on here, but i would like to  kindly ask you how do i make these changes?? I want runelite to show offline friends on minimap badly and i'm going crazy trying to figure out how to do it...

Comment (Nightfirecat) [reply runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsOverlay.java]: This is a feature rejected by the team. We will not be assisting you with making such changes.

**Pull request closed without merging**